server/routes.go

```diff
@@ -1190,12 +1192,12 @@ func Serve(ln net.Listener) error {
 		srvr.Close()
 		schedDone()
 		sched.unloadAllRunners()
-		gpu.Cleanup()
+		runners.Cleanup(build.EmbedFS)
 		done()
 	}()
 
-	if err := llm.Init(); err != nil {
-		return fmt.Errorf("unable to initialize llm library %w", err)
+	if _, err := runners.RefreshDir(build.EmbedFS); err != nil {
```

Comment:
Better however the `Dir` suffix still sounds like it would return a dir – maybe just `Refresh` ?